extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java

```diff
@@ -227,6 +235,20 @@ public static Throwable extractRootCause(Throwable throwable) {
             }
             return throwable;
         }
+
+        public static void markIfOtherAuthFailure(RoutingContext event, Throwable throwable) {
+            if (!(throwable instanceof AuthenticationException)) {
+                event.put(OTHER_AUTH_FAILURE, Boolean.TRUE);
+            }
+        }
+
+        public static void removeMarkAsOtherAuthFailure(RoutingContext event) {
+            event.remove(OTHER_AUTH_FAILURE);
+        }
+
+        public static boolean isOtherAuthFailure(RoutingContext event) {
```

Comment:
I'd not mind to have `Auth` expanded to `Authentication` just to make these names a bit more descriptive, but it is not a big deal

Reply:
sure, done